LICENSE

```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2018, The Scikit-HEP Project
+Copyright (c) 2018-2019, The particle developers.
```

Comment:
I believe we can just put our names here. I have permission to do so, anyway.

Reply:
OK, will do.